### PR TITLE
S03 us18 1 compter le nombre de vues par video

### PR DESCRIPTION
**Pull request by l00ma**

### CODE REVIEW

**Review by val-fr69 — APPROVED** [review]

File: env.test.local [hunk 1]
```diff
@@ -0,0 +1 @@
+DATABASE_URL="mysql://harry:potter@localhost:3306/projet3?serverVersion=8.0.32&charset=utf8mb4"
```
Comment (aldaitzwild) [inline env.test.local:1]: It is nice of you to show publicly your DB credentials ... but maybe you should'nt

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: env.test.local (hunk 1 above)
Comment (l00ma) [reply env.test.local:1]: ooops !

File: src/Repository/ViewedRepository.php [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+    //    public function findOneBySomeField($value): ?Viewed
+    //    {
+    //        return $this->createQueryBuilder('v')
+    //            ->andWhere('v.exampleField = :val')
+    //            ->setParameter('val', $value)
+    //            ->getQuery()
+    //            ->getOneOrNullResult()
+    //        ;
+    //    }
```
Comment (maxdlr) [inline src/Repository/ViewedRepository.php:75]: Commented code

File: templates/shared/thumbnail/big-thumbnail.html.twig [hunk 3]
```diff
@@ -16,29 +16,29 @@
...
+									<i class='bt-watch bi bi-clock text-light la{{ item.id }}' title="ajouter à voir plus tard"></i>
 								{% endif %}
 							</div>
 						</a>
 						<a class="like" href="{{ path('like_add', { idVideo: item.id }) }}">
 						{% set isLiked = app.user.likedVideos|filter(video => video.id == item.id )|first %}
 						<div class="bt-social-btn">
 							{% if isLiked %}
-							<i class='bt-like bi bi-hand-thumbs-up-fill text-secondary li{{ item.id }}'></i>
+							<i class='bt-like bi bi-hand-thumbs-up-fill text-secondary li{{ item.id }}' title="ne pas liker"></i>
```
Comment (maxdlr) [inline templates/shared/thumbnail/big-thumbnail.html.twig:39]: "unlike" peut etre ?

File: tests/RoutingTest.php [hunk 4]
```diff
@@ -22,9 +22,11 @@ public function urlProvider(): mixed
     {
         yield ['/'];
         yield ['/video/show/1'];
+        yield ['/video/add/1'];
         yield ['/search/dota'];
         yield ['/search/cs'];
         yield ['/search/overwatch'];
         yield ['/upload'];
+        yield ['/viewed/1'];
     }
```
Comment (maxdlr) [inline tests/RoutingTest.php:31]: Ajouter /login et /register ?

**Review by maxdlr — CHANGES_REQUESTED** [review]

File: src/Repository/ViewedRepository.php (hunk 2 above)
Comment (l00ma) [reply src/Repository/ViewedRepository.php:75]: This code is generated by symfony when creating repositories. 
You will find same thing in each other repositories.
So nothing to delete here

File: templates/shared/thumbnail/big-thumbnail.html.twig (hunk 3 above)
Comment (l00ma) [reply templates/shared/thumbnail/big-thumbnail.html.twig:39]: unlike is english word. 
"unliker" seems not appropriate

File: tests/RoutingTest.php (hunk 4 above)
Comment (l00ma) [reply tests/RoutingTest.php:31]: all these routes are added now:
        yield ['/'];
        yield ['/video/show/1'];
        yield ['/video/add/1'];
        yield ['/search/dota'];
        yield ['/search/cs'];
        yield ['/search/overwatch'];
        yield ['/upload'];
        yield ['/viewed/1'];
        yield ['/fav/1'];
        yield ['/later/1'];
        yield ['/like/1'];
        yield ['/login'];
        yield ['/logout'];
        yield ['/admin'];
        yield ['/register'];
        yield ['/verify/email'];

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**